Lombiq.Hosting.Tenants.MediaStorageManagement.Tests.UI/Extensions/MediaStorageManagementExtensions.cs

```diff
@@ -16,7 +16,7 @@ public static class MediaStorageManagementExtensions
             {
                 argumentsBuilder
                     .AddWithValue(
-                        "OrchardCore:Lombiq_Hosting_Tenants_MediaStorageManagement:MaximumStorageQuotaBytes",
+                        "OrchardCore:Default:Lombiq_Hosting_Tenants_MediaStorageManagement:MaximumStorageQuotaBytes",
```

Comment:
Why is `Default` here necessary?

Reply:
Otherwise, the new setting for demonstrating the module will override this one because it is a tenant level setting.

Reply:
Add a comment about this here then, please.